backend/api/oauthEnabledClient.ts

```diff
@@ -118,7 +118,7 @@ export class OauthApiClient {
    * @param body
    * @returns A Promise which resolves to the superagent result object, or the superagent error object if it is rejected
    */
-  post = <T>(context: ClientContext, path: string, body: never): Promise<ClientResponse<T>> => {
+  post = <T>(context: ClientContext, path: string, body: any): Promise<ClientResponse<T>> => {
```

Comment:
you normally _do_ want to supply a body to a POST endpoint

Reply:
it's ok to not include a body when `any` is asked for. but you cannot pass _any_ body if it requires `never`!

you can still call `client.post(context, '/…', undefined)` on new version cos an `undefined` counts as an `any`.

Reply:
Ah OK, so the original was wrong but this makes `POST` requests' body optional which is what you want 👌